core/templates/services/user.service.ts

```diff
@@ -73,7 +73,7 @@ export class UserService {
     }
 
     async setProfileImageDataUrlAsync(
-        newProfileImageDataUrl: string): Promise<PreferencesBackendDict> {
+        newProfileImageDataUrl: string): Promise<Object> {
```

Comment:
Why this change?

Reply:
this was originally wrong, i checked the response, it was a object

Reply:
We don't usually allow Object as a type (because it is too generic). Can you check if the Object is dictionary like or is it String. (Instances of Class string are shown as Objects).

Reply:
fixed